### PR TITLE
Add link to Mega-Linter (including rstcheck)

### PR DESCRIPTION
**Pull request by nvuillam**
I added your nice tool in [Mega-Linter](https://nvuillam.github.io/mega-linter) :)

Please tell me if any information could be arranged on [rstcheck](https://nvuillam.github.io/mega-linter/descriptors/rst_rstcheck/) page :)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -268,6 +267,11 @@ Add this to your ``.pre-commit-config.yaml``
         hooks:
         -   id: rstcheck
 
+Use with Mega-Linter:
+=====================
+
+Just install `Mega-Linter <https://nvuillam.github.io/mega-linter/>`__ in your repository, `rstcheck <https://nvuillam.github.io/mega-linter/descriptors/rst_rstcheck/>`__ is part of the 70 linters activated out of the box
```
Comment (Cielquan) [inline README.rst:273]: Please add line breaks and a period like so:

```rst
Just install `Mega-Linter <https://nvuillam.github.io/mega-linter/>`__ in your repository,
`rstcheck <https://nvuillam.github.io/mega-linter/descriptors/rst_rstcheck/>`__
is part of the 70 linters activated out of the box.
```

**Review by Cielquan — CHANGES_REQUESTED** [review]

**Comment by Cielquan** [comment]
The docs over at MegaLinter are no longer up-to-date with the most recent releases:
https://megalinter.github.io/latest/descriptors/rst_rstcheck/#readme

**Comment by nvuillam** [comment]
@Cielquan done :)

**Comment by Cielquan** [comment]
LGTM

**Review by Cielquan — APPROVED** [review]

**Comment by Cielquan** [comment]
Got a merge conflict because of my update of the README.

**Comment by Cielquan** [comment]
Thanks for the contribution. :)

**Comment by nvuillam** [comment]
Thanks for accepting the PR , have a nice day :)

**Pull request merged**